src/scripts/main.js

```diff
@@ -1,5 +1,48 @@
 'use strict';
 
-document.addEventListener('click', e => {
-  // write code here
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+wall.addEventListener('click', e => {
+  const rectangle = e.target.getBoundingClientRect();
+  const x = e.clientX - rectangle.left
+    - parseInt(getComputedStyle(wall).borderLeftWidth);
+  const y = e.clientY - rectangle.top
+    - parseInt(getComputedStyle(wall).borderTopWidth);
```

Comment:
read about clientHeight and clientWidth
https://learn.javascript.ru/size-and-scroll#clientwidth-height